recipes/mss/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
```

Comment:
It should :smile: I computed the checksum for version `1.1.0`. If you are updating it to `1.2.0` you have to update the `sha256` too. (Sorry, that is an extra step, but ensures that the tarball is correct.)